common/scripts/actionslist.lua

```diff
@@ -0,0 +1,31 @@
+function addEntry(sType, bFocus)
```

Comment:
Probably should have the license header for consistency. Also, spaces to tabs.

Reply:
Added the license.